README.md

```diff
@@ -0,0 +1,40 @@
+# Excalidraw
+
+Excalidraw is a whiteboard tool that lets you easily sketch diagrams.
+
+## Try it now
+
+If you want to start sketching, please check https://excalidraw.com
```

Comment:
I would rephrase this using an active sentence: Go to https://excalidraw.com to start sketching